designer/client/src/components/modals/ValidationLabels.tsx

```diff
@@ -13,6 +14,8 @@ type Props = {
 type ValidationError = {
```

Comment:
To me, the type should be like

type Params = OperationMismatchParams | SomethingDifferentParams;    
type ErrorCode =  'OPERATOR_MISMATCH' | SomethingDifferent

type ValidationError = {
description: string;     
params: Params
errorCode: ErrorCode;
}